otodom/task_1/Valentyn/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+	"base_url": "str",
+	"price_min": "str",
+	"price_max": "str",
+	"city": "str",
+	"property_type": "str",
+	"only_for_sale": "bool",
+	"only_for_rent": "bool"
+}
```

Comment:
these where meant to be loaded into program and based on them, the url should be constructed.

Reply:
Did in the latest commit